orchestration/scripts/clingen-temp-table-pk-deletes.sh

```diff
@@ -0,0 +1,48 @@
+set -euo pipefail
+
+# Copy-pasted from StackOverflow.
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+# Point to BQ metadata we expect to be present on disk.
+declare -r TABLE_DIR=/bq-metadata/${TABLE}
+declare -r PK_COLS=$(cat ${TABLE_DIR}/primary-keys)
+
+# Build the WHERE clause of the SQL query.
+declare -a DATAREPO_COLUMNS=()
+declare -a COMPARISONS=()
+
+# Building one array for columns to select and one for selection criteria
+for c in ${PK_COLS//,/ }; do
+  DATAREPO_COLUMNS+=("datarepo_${c} as ${c}")
+  # where prefixed version is not null and nonprefixed version is null
+  COMPARISONS+=("datarepo_${c} is not null and ${c} is null")
```

Comment:
I might be getting confused here, but why alias `datarepo_${c} as ${c}`? Wouldn't that make it impossible to compare them below in `datarepo_${c} is not null and ${c} is null`?

Reply:
Aliasing executes after filters, so it won't cause problems with the logic

Reply:
Ohh, that makes sense, thank you!